tests/end_to_end/netsh_test.cpp

```diff
@@ -1091,3 +1091,107 @@ TEST_CASE("show processes", "[netsh][processes]")
         REQUIRE(result == NO_ERROR);
     }
 }
+
+#if !defined(CONFIG_BPF_JIT_DISABLED) || !defined(CONFIG_BPF_INTERPRETER_DISABLED)
```

Comment:
The notion of pin/unpin should not be limited to just JIT/interpreted, but should also work with native. I'd like to see tests with .sys/.dll files too.

Reply:
Other netsh tests are using the same approach - JIT only

Reply:
This is issue #4261.